projects/extension/tests/vectorizer/test_vectorizer.py

```diff
@@ -260,37 +266,6 @@ def test_vectorizer_timescaledb():
             actual = cur.fetchone()[0]
             assert actual == 3
 
-            # bob should have select on the source table
```

Comment:
why were these deleted?

Reply:
This was done in the commits I based my changes onto and explained here:
https://github.com/timescale/pgai/pull/407#discussion_r1942965198

Apparently those assertions are done twice right now.

Reply:
yes, those tests are now redundant.